shared/common-adapters/back-button.native.js

```diff
@@ -19,10 +20,12 @@ export default class BackButton extends Component {
 
   render () {
     return (
-      <Box style={{...styles.container, ...this.props.style}} onClick={e => this.onClick(e)}>
-        <Icon type='fa-arrow-left' style={styles.icon}/>
-        <Text type='BodyPrimaryLink' onClick={e => this.onClick(e)}>Back</Text>
-      </Box>
+      <TouchableWithoutFeedback onPress={e => this.onClick(e)}>
+        <Box style={{...styles.container, ...this.props.style}} >
```

Comment:
I got an error when doing this with TouchableHighlight that Box should forward `setNativeProps`, do you not get that error here?

I basically changed Box to be
```
import {View} from 'react-native'
export default View
```